cs3/app/provider/v1beta1/resources.proto

```diff
@@ -0,0 +1,46 @@
+// Copyright 2018-2019 CERN
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// In applying this license, CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+syntax = "proto3";
+
+package cs3.app.provider.v1beta1;
+
+option csharp_namespace = "Cs3.App.Provider.V1Beta1";
+option go_package = "providerv1beta1";
+option java_multiple_files = true;
+option java_outer_classname = "ResourcesProto";
+option java_package = "com.cs3.app.provider.v1beta1";
+option objc_class_prefix = "CAP";
+option php_namespace = "Cs3\\App\\Provider\\V1Beta1";
+
+// Represents the information for the app URL to be called by the clients.
+message OpenInAppURL {
+  // REQUIRED.
+  // The URL that clients will perform requests to.
+  string app_url = 1;
+  // REQUIRED.
+  // The method for the request to be made.
+  string method = 2;
```

Comment:
Looks all good to me. Do you think we need to restrict this to either `GET` or `POST`?

Reply:
Yep, makes sense. I'll add a comment